internal/checker/printer.go

```diff
@@ -343,20 +315,20 @@ func (c *Checker) valueToString(value any) string {
 	return ValueToString(value)
 }
 
-func (c *Checker) WriteSymbol(symbol *ast.Symbol, enclosingDeclaration *ast.Node, meaning ast.SymbolFlags, flags SymbolFormatFlags, writer printer.EmitTextWriter) string {
-	return c.symbolToStringEx(symbol, enclosingDeclaration, meaning, flags, writer)
+func (c *Checker) WriteSymbol(symbol *ast.Symbol, enclosingDeclaration *ast.Node, meaning ast.SymbolFlags, flags SymbolFormatFlags) string {
+	return c.symbolToStringEx(symbol, enclosingDeclaration, meaning, flags)
 }
 
-func (c *Checker) WriteType(t *Type, enclosingDeclaration *ast.Node, flags TypeFormatFlags, writer printer.EmitTextWriter) string {
-	return c.typeToStringEx(t, enclosingDeclaration, flags, writer)
+func (c *Checker) WriteType(t *Type, enclosingDeclaration *ast.Node, flags TypeFormatFlags) string {
+	return c.typeToStringEx(t, enclosingDeclaration, flags)
```

Comment:
Just get rid of these `WriteXXX` functions now? They're just needless wrappers.

Reply:
I can but then everything they call has to be exported. I guess I can make that change.